packages/react-components/react-checkbox/src/components/Checkbox/useCheckboxStyles.ts

```diff
@@ -143,6 +141,23 @@ const useInputStyles = makeStyles({
       },
     },
   },
+
+  before: {
+    right: 0,
+  },
+  after: {
+    left: 0,
+  },
```

Comment:
Could you double check that this still works correctly in RTL? It might be worthwhile to add some RTL variants for some Checkbox screener tests.

Reply:
I've added some RTL VR tests to verify this.